### ISSUE TITLE
Show technical names on page load not working / saving

### ISSUE THREAD

**Issue by SteveMunsell** [arnoudkooi/ServiceNow-Utils#231]
This is a more recent issue I've run into. I have even gone as far as removing and re-adding the extension but the behavior is the same.

When I go into the settings and select the  "Show technical names on page load" option it does not save that option and when I reload the page the technical names are not shown. Going back into the settings the box is unchecked every time. 

Additionally, the option above it "Enable Code Editor buttons (Monaco Code Editor)" is always checked, it does the opposite and will not uncheck. Was there a save button added somewhere because I never noticed having to save the settings before.

I have tested and confirmed this issue in Brave and Chrome, both do the same thing as far as I can tell.

[image](https://user-images.githubusercontent.com/20819396/167471008-cec9b407-4610-457e-baa4-661b24d456d9.png)

**Comment by arnoudkooi** [comment]
Maybe join a quick call? https://servicenow.zoom.us/my/arnoudkooi

**Comment by arnoudkooi** [comment]
Can you run this code in the console, for example in the **code search** window. (run slashcommand `/ code` to open)
Not in the normal browser window

```
chrome.storage.sync.get("snusettings", obj => { 
    obj.snusettings.slashcommands = '';
    console.log(JSON.stringify(obj,4,4)) 
} );
```

If you paste the result here, the result should be something like below
<img width="584" alt="image" src="https://user-images.githubusercontent.com/840781/167481636-9e547289-11f4-43ca-833c-b49f050a732c.png">

We could also set up a quick call to troubleshoot

**Comment by arnoudkooi** [comment]
```
chrome.storage.sync.get("snusettings", obj => { 
    console.log(JSON.stringify(obj,4,4)) 
} );
```

and like this?

do you have time to hop on a quick call?

**Comment by SteveMunsell** [comment]
This is the result of the above code. If you'd like you can also text me directly on my google number and we can setup a call as well ‪(727) 265-1403‬

![image](https://user-images.githubusercontent.com/20819396/167649567-2bf4cd9e-2195-419e-b147-1c057eb448d2.png)

**Comment by SteveMunsell** [comment]
This was fixed, I went ahead and did a third re-install of the plugin and for some reason that was the one that made it work like normal. My best guess is that in the ext properties I didn't have "Allow Access" turned on to my SN instance and had it set to ask every time. Thanks for the info and the amazing tool! ServiceNow wouldn't be the same without this and I love it so much so that I will be taking a class on it during Knowledge 2022 so keep up the amazing work.
